plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-typography.tsx

```diff
@@ -201,18 +202,16 @@ export const SidebarNavigationScreenTypography = () => {
 										</Button>
 										<Button
 											onClick={ async () => {
-												await dispatch(
-													OPTIONS_STORE_NAME
-												).updateOptions( {
```

Comment:
Prior to this change, we were indirectly triggering synchronous font installation by triggering the state whenever accessing the link to the assembler. Now, you created a dedicated event for this, which is great! With that said, can you please re-add this async font installation here? It's important to have **both the sync and async fonts installation as a safety net** in case one of them fails to execute for some reason.

Reply:
> Prior to this change, we were indirectly triggering synchronous font installation by triggering the state whenever accessing the link to the assembler. Now, you created a dedicated event for this, which is great! With that said, can you please re-add this async font installation here? It's important to have both the sync and async fonts installation as a safety net in case one of them fails to execute for some reason.

I just moved the logic [here](https://github.com/woocommerce/woocommerce/pull/45580/files#diff-10fabebab9142a466545a1dc81f748b9cbf82bff2fc280e5aee18e1629a0c900R238-R248)